src/Microsoft.SqlTools.Credentials/Localization/sr.pt-BR.resx

```diff
@@ -0,0 +1,135 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
BR here is upper case, is it not going to cause issues?

Reply:
I just spotted a few other places where the BR in file name is in upper case.

Reply:
That shouldn't cause issues with this stuff since we tend to just glob all resx files, but yeah being consistent is good.